generators/base/generator-base.mjs

```diff
@@ -288,11 +282,7 @@ export default class JHipsterBaseGenerator extends PrivateBase {
    * @return {boolean} true if the Authority is built-in.
    */
   isUsingBuiltInAuthority() {
```

Comment:
Only used https://github.com/jhipster/generator-jhipster/blob/c4223686a9457e99c50a41584b950f3cafa8e23c/generators/server/generator.mjs#L303.
Same as the isUsingBuiltInUser

Reply:
Which is important ^^

Reply:
Not as easy at it seems:
Ideally it should be moved to `base-app` generator right? BUT it's used by the `isBuiltInAuthority(entityName)` of the `entity` generator (and in some templates), which doesn't inherit from this one (it inherits form base).
I agree that it should be done, but generator inheritance should be thought and reworked as a prerequisite (And I definitely don't feel confident to achieve it by myself)

Reply:
After a lot of try, I won't try to move the methods from generator to others before moving the code: there are too much interleaving and fully externalizing first will definitely help

Reply:
Let me do it. I’ve already started.
We will need to split the user/userManagement logic.